### PR TITLE
Wdi16

### PR DESCRIPTION
**Pull request by superbuggy**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -77,10 +77,27 @@ user = {
...
+    <li> The syntax for referencing properties differs.
+      <ul>
+        <li>JS: <code>user.name</code> or <code>user["name"]</code> </li>
+        <li>Ruby: <code>user[:name]</code> </li>
+      </ul>
+    </li>
+    <li>.push, .pop, .join</li>
+    <li>  https://ruby-doc.org/core-2.2.3/Enumerable.html </li>
+  </ol>
+</details>
```
Comment (nayana487) [inline readme.md]: ~~^^ In the details summary for the answer to number 4, it'd look nicer to have the text "Ruby Documentation" and then hyperlink~~

edit: fixed. jk 👍

**Comment by nayana487** [comment]
All the snippets below have the exact same result:
 
 ```
 Alice
 Bob
 Carol
``` 

what snippets?

File: readme.md [hunk 2]
```diff
@@ -305,7 +325,12 @@ Use `each` to do the following...
   flavors = [ "vanilla", "chocolate", "strawberry", "butter pecan", "cookies and cream", "rainbow" ]
   toppings = [ "gummi bears", "hot fudge", "butterscotch", "rainbow sprinkles", "chocolate sprinkles" ]
   ```
-Hint: Use nested enumerables or check out [product](http://apidock.com/ruby/Array/product)
+<details>
+  <summary>
+    Hint
+  </summary>
+  Use nested enumerables or check out [product](http://apidock.com/ruby/Array/product)
+</details>
```
Comment (nayana487) [inline readme.md]: The hyperlink for "product" didn't stick here

File: readme.md (hunk 1 above)
Comment (superbuggy) [reply readme.md]: 👍

**Comment by superbuggy** [comment]
Thanks Nayana!

**Pull request merged**